__test__/main.spec.ts

```diff
@@ -98,12 +99,20 @@ describe('reflink', () => {
     const dir = sandboxDir();
     await prepare(dir);
 
-    expect(() => {
+    try {
       reflinkFileSync(
         join(dir, 'file-does-not-exist.txt'),
         join(dir, 'file1-copy.txt')
       );
-    }).toThrow();
```

Comment:
why not using `await expect(...).rejects.toMatchObject({`?

Reply:
I have replace all `try catch` block with `expect().rejects` (https://github.com/pnpm/reflink/pull/11/commits/bcbbfaaa822f3fbab2e606901b4eab649b7ec563). However, I leave the other `toThrow` alone because I didn't intend to touch them in the first place. I didn't want to touch them because there could be difference between Windows and POSIX in whether the error code would be `EEXIST` or not.